docs/tutorials/cloud-foundry.md

```diff
@@ -0,0 +1,730 @@
+---
+layout: default
+title: "Migrating Cloud Foundry apps to Kubernetes"
+permalink: /tutorials/cloud-foundry/
+parent: Tutorials
+nav_order: 4
+---
+
+# Migrating and deploying Cloud Foundry applications to Kubernetes
+
+## Description
+
+This document takes us through the steps that will install Move2Kube and use Move2Kube's 3 step process (collect, plan and transform) to create deployment artifacts for Cloud Foundry apps. It also takes through the process to customize for a specific cluster. Here, we are going to use the data from [samples/cloud-foundry](https://github.com/konveyor/move2kube-demos/tree/main/samples/cloud-foundry).
+
+
+## Prerequisites
+
+1. Install Move2Kube.
+
+   ```console
+   $ MOVE2KUBE_TAG='v0.3.0-rc.0' bash <(curl https://raw.githubusercontent.com/konveyor/move2kube/main/scripts/install.sh)
+   ```
+
+2. Install dependencies.
+  * [Docker](https://www.docker.com/get-started)
+  * [operator-sdk](https://docs.openshift.com/container-platform/4.1/applications/operator_sdk/osdk-getting-started.html#osdk-installing-cli_osdk-getting-started)
```

Comment:
We are no longer creating operators. So this dependency no longer holds

Reply:
I have removed the operator-sdk from the dependencies and added Podman as an alternative to Docker.